packages/amplication-server/src/core/github/github.service.ts

```diff
@@ -102,13 +102,22 @@ export class GithubService implements IGitClient {
       parseInt(data.installationId)
     );
 
+    const installation = await octokit.rest.apps.getInstallation({
+      // eslint-disable-next-line @typescript-eslint/naming-convention
+      installation_id: parseInt(data.installationId)
+    });
+
+    if (installation.data.account.type.toLowerCase() === 'user') {
+      return null;
```

Comment:
I think we should throw an "Unsupported Operation Exception" or something similar 
Or - at least add a comment that creating repo for personal account is not support with a link to a new GitHub Issue